Description: Fix FTBFS with clang.
  Fixed "wrong main declaration" error in ann2fig/ann2fig.cpp
Author: Arthur Marble <arthur@info9.net>
Bug-Debian: http://bugs.debian.org/748817
Reviewed-by: Anton Gladky <gladk@debian.org>
Last-Update: 2014-05-31

--- a/ann2fig/ann2fig.cpp
+++ b/ann2fig/ann2fig.cpp
@@ -580,8 +580,10 @@ void readANN()
 // procedure.
 //----------------------------------------------------------------------
 
-main(int argc, char **argv)
+int main(int argc, char **argv)
 {
 	getArgs(argc, argv);						// get input arguments
 	readANN();									// read the dump file
+
+	return 0;
 }
